fre/pp/configure_script_yaml.py

```diff
@@ -12,6 +12,30 @@
 import click
 import metomi.rose.config
 
+def yamlloads(yaml, experiment):
+    """
+    Load the yaml that holds experiment yamls.
+    EXtract FRE properties defined in the yaml.
+    """
+    with open(yaml,"r") as f:
+        y=yaml.safe_load(f)
+
+    # Define properties (FRE properties)
+    stem=y.get("stem")
+
+    for propkey,propvalue in y.items():
+        if propkey == "exp":
+            for i in propvalue:
+                # if the experiment listed in the click options matches 
+                # an experiment name listed in the main yaml, return 
+                # that experiment yaml
+                if i.get("expname") == experiment:
+                    # Load this specific pp yaml for rose suite and whatnot
+                    expyaml=i.get("ppyaml")
+
+    # return experiment yaml and defined FRE properties 
+    return(expyaml,stem)
+
```

Comment:
This is great! FRE Property expansion happens here.

I think there has to be special case expansions for the command-line arguments: experiment, platform, and target: $(name), $(platform), and $(target).

For the rest, it would be ideal to have them be completely user-specified, and not limited to a particular set, e.g. "$(stem).